checks/raw/pinned_dependencies_test.go

```diff
@@ -1442,3 +1447,226 @@ func countUnpinned(r []checker.Dependency) int {
 
 	return unpinned
 }
+
+func stringAsPointer(s string) *string {
+	return &s
+}
+
+func boolAsPointer(b bool) *bool {
+	return &b
+}
+
+// TestCollectDockerfilePinning tests the collectDockerfilePinning function.
+func TestCollectDockerfilePinning(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name                string
+		filename            string
+		outcomeDependencies []checker.Dependency
+		expectError         bool
+	}{
+		{
+			name:        "Pinned dockerfile",
+			filename:    "./testdata/.github/workflows/github-workflow-download-lines.yaml",
+			expectError: true,
+		},
```

Comment:
is this a copy/paste error in name? We pass in a github workflow, and get an error? so the name should reflect that?

Reply:
Fixed in https://github.com/ossf/scorecard/pull/3632/commits/8b8d0877aef2466aa17dad441203798b579fa5a2